docs/design/99-error-handling.md

```
@@ -0,0 +1,75 @@
+
+1. `$XDG_DATA_HOME/shellbar/` （通常は `~/.local/share/shellbar/`）
+2. `$HOME/.local/share/shellbar/` （XDG_DATA_HOMEが未設定の場合）
+
+### ログファイル名
+
+- ファイル名：`shellbar.log`
+- ローテーション：サイズまたは日付ベース（将来実装）
+
+### ログフォーマット
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)